src/pages/søknad/steg/inntekt/Inntekt.tsx

```diff
@@ -24,16 +24,15 @@ type FormData = SøknadState['inntekt'];
 const trygdeytelserIUtlandetSchema = yup.object({
     beløp: yup
         .number()
-        .typeError('Beløp må være et tall')
+        .typeError('Den lokale valutaen må være et tall')
         .positive()
-        .label('Beløp')
-        .required() as yup.Schema<unknown> as yup.Schema<string>,
-    type: yup.string().required(),
-    valuta: yup.string().required(),
+        .required('Fyll ut hvor mye du får i lokal valuta') as yup.Schema<unknown> as yup.Schema<string>,
+    type: yup.string().required('Fyll ut hvilken type ytelsen er'),
+    valuta: yup.string().required('Fyll ut valutaen for ytelsen'),
 });
 
 const schema = yup.object<FormData>({
-    harForventetInntekt: yup.boolean().nullable().required(),
+    harForventetInntekt: yup.boolean().nullable().required('Fyll ut om du forventer arbeidsinntekt'),
```

Comment:
Ref samme kommentar for ektefelle. Det er vel ikke søkeren som forventer dette, men heller nav.

Reply:
spørsmåls formulering er 'Forventer du å ha arbeidsinntekt fremover?'. Er det ikke søkeren da?